sources/web/datalab/polymer/components/datalab-data/datalab-data.html

```diff
@@ -61,64 +51,61 @@
         width: 20px;
         padding-right: 5px;
       }
-      .collections-container {
-        height: 100%;
+      .toolbar-spacer {
+        flex-grow: 1;
+      }
+      #data-container {
         display: flex;
+        margin: 15px;
+      }
+      .resultsColumn {
+        flex-grow: 1;
       }
-      .tables-container {
+      .results-container {
         height: 100%;
         display: flex;
       }
-      #collections {
-        flex-grow: 1;
+      #results {
+        width: 100%;
       }
-      #tables {
-        flex-grow: 1;
+      #detailsPane {
+      }
+      .details-pane-enabled--true {
+        width: 400px;
+      }
+      .details-pane-enabled--false {
+        width: 0px;
       }
     </style>
 
     <!--Toolbar with action buttons -->
     <div id="toolbar">
-      <paper-button class="toolbar-button" on-click="_search">
-        <iron-icon icon="search"></iron-icon>
-        <span>Search</span>
+      <div class="toolbar-spacer"></div>
+      <paper-button id="toggleDetails" class="navbar-button" on-click="_toggleDetailsPane">
+        <iron-icon icon="visibility"></iron-icon>
       </paper-button>
     </div>
 
-    <!-- Multi-column layout: collections (BQ datasets), tables -->
-    <div class="tableDrillDown">
-
-      <!-- Collections (BQ datasets) -->
-      <div class="collectionsColumn filteredColumn">
-        <div>
-          Collections Filter:
-            <paper-input id="collectionsFilterBox" label="Optional filter value" spellcheck=false autofocus
-                         value="{{_collectionsFilterValue}}" no-label-float>
-            </paper-input>
+    <div id="data-container">
+      <div class="resultsColumn">
+        <!-- Input box to enter search terms -->
+        <div id="search-div">
+          <iron-a11y-keys id="searchKeys" keys="enter" on-keys-pressed="_search"></iron-a11y-keys>
```

Comment:
Better yet, we can fire after a delay, say 300ms, so that we buffer the user's keystrokes, instead of triggering service calls on each key. I'd add this as a TODO since it's just an improvement.